### DISCUSSION TITLE
.NET MAUI 9.0.12 (Service Release 1.2)

### DISCUSSION THREAD

**Discussion by davidortinau** [dotnet/maui#26351] [Announcements]
This release is now available via [NuGet.org](https://www.nuget.org/packages/Microsoft.Maui.Controls/9.0.12) and is compatible with the latest .NET 9 workload. 

```
<ItemGroup>
  <PackageReference Include="Microsoft.Maui.Controls" Version="9.0.12" />
  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="9.0.12" />
</ItemGroup>
```

- [Release notes](https://github.com/dotnet/maui/releases/tag/9.0.12)

**Comment by StuRamsey** [comment]
Any ideas why VS2022 NuGet Manager wants to update these two packages into just created .Net 8 Maui project, because they cause it to crash.
**The type or namespace name 'MauiAppCompatActivity' could not be found**
It's okay before updating, nothing done to the project, just create then go straight to NuGet Manager to make sure everything's up to date before I start my first go with Maui and this happens :-(
